batch7crud-roll10/src/main/java/com/lftechnology/batch7crud/validator/StudentValidator.java

```diff
@@ -0,0 +1,36 @@
+package com.lftechnology.batch7crud.validator;
+
+import com.lftechnology.batch7crud.constant.EntityConstant;
+import com.lftechnology.batch7crud.entity.Student;
+import sun.util.resources.cldr.de.CalendarData_de_LI;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'sun.util.resources.cldr.de.CalendarData_de_LI'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AUselessImportCheck)
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Replace this usage of Sun classes by ones from the Java API. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1191)